metasploit-framework.gemspec

```diff
@@ -203,4 +203,8 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'aws-sdk-s3'
   spec.add_runtime_dependency 'aws-sdk-ec2'
   spec.add_runtime_dependency 'aws-sdk-iam'
+
+  # Needed for WebSocket Support
+  spec.add_runtime_dependency 'faye-websocket'
+  spec.add_runtime_dependency 'eventmachine'
```

Comment:
Are there any websocket implementations that don't need eventmachine as a dependency? We use it in Metasploit Pro, but I'm not 100% sure it'll work on ARM and all the OSes that Metasploit Framework works on today.

Reply:
@jmartin-r7 did some testing and looks like it will work out on ARM.

Reply:
Are we good here?